packages/webpack/src/builder.js

```diff
@@ -57,6 +57,10 @@ export class WebpackBundler {
       webpackConfigs.push(this.getWebpackConfig('Server'))
     }
 
+    const additionalConfigs = []
+    await this.buildContext.nuxt.callHook('build:config', additionalConfigs)
```

Comment:
Why not just pass webpackConfigs? And also a ref to the builder?

For hacking purposes it would be interesting to just have access to everything.

Reply:
We've already provided `extend` for customising, I'm wondering it we should expose here again.

Reply:
Probably we can use this hook to implement extend directly

Reply:
I added webpackConfigs for now, let's see further comments